### PR TITLE
Readonly gateway API fix

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: core/commands/root.go [hunk 1]
```diff
@@ -132,14 +140,17 @@ var rootROSubcommands = map[string]*cmds.Command{
...
+	"refs": RefsROCmd,
 	//"resolve": ResolveCmd,
 }
 
-var RootRO = &cmds.Command{}
-
 func init() {
 	*RootRO = *Root
+
+	// sanitize readonly refs command
```
Comment (rht) [inline core/commands/root.go]: Or it is safe (and actually useful) to expose local refs.

Comment (jbenet) [inline core/commands/root.go]: i think it is useful, and also that safety depends on context. keep it disabled until we find a strong reason to enable it?

**Comment by jbenet** [comment]
@rht also: we forgot to try running _write_ commands and checking they fail

File: core/commands/root.go (hunk 1 above)
Comment (rht) [inline core/commands/root.go]: so be it

**Comment by rht** [comment]
> @rht also: we forgot to try running write commands and checking they fail

(just made this while offline)

**Pull request merged**